docs/users-guide/dashboard-subscriptions.md

```diff
@@ -46,6 +46,10 @@ For Slack subscriptions, you can set up a subscription for a channel (like #gene
 
 You can specify how often Metabase sends a Slack message (hourly, daily, weekly, or monthly), and whether to send a message if the dashboard fails to return results.
 
+## Enterprise Edition: customize filter values for each dashboard subscription
+
+Metabase Enterprise Edition allows you to [set up multiple subscriptions for the same dashboard](../enterprise-guide/dashboard-subscriptions.md), so you can send different emails or Slack messages to different groups of people, and with different filter values applied.
```

Comment:
> Metabase Enterprise Edition allows you to [set up multiple subscriptions for the same dashboard]

Again, this is possible in OSS — see this screenshot from Stats:

![image](https://user-images.githubusercontent.com/2223916/114080137-df43a700-985f-11eb-9db1-9430e03e0112.png)